oscarlab/graphene
# Pylint fixes, January 2020

## Affected components


## Description of the changes 
Style was changed for two several python scripts. No functional change.

## How to test this PR? 
Test any use of the tools in question. Also check for no pylint errors for those scripts (should be done automatically).

fixes #1239 
cc @yamahata


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1241)
